python/morpheus/morpheus/stages/inference/identity_inference_stage.py

```diff
@@ -44,18 +44,17 @@ def __init__(self, inf_queue: ProducerConsumerQueue, c: Config):
         self._max_batch_size = c.model_max_batch_size
         self._seq_length = c.feature_length
 
-    def calc_output_dims(self, x: MultiInferenceMessage) -> typing.Tuple:
-        return (x.count, self._seq_length)
+    def calc_output_dims(self, msg: ControlMessage) -> typing.Tuple:
+        return (msg.tensors().count, self._seq_length)
```

Comment:
```suggestion
    def calc_output_dims(self, msg: ControlMessage) -> tuple[int, int]:
        return (msg.tensors().count, self._seq_length)
```

Reply:
Not really a part of this PR, but something I'm thinking about is if `msg.tensors().count` is a common pattern, calling `msg.tensors()` might require a bit more cost than we think, we might want to consider a `tensor_count` property.